src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/StringUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.utils;
+
+import java.util.Locale;
+
+public class StringUtils {
+
+    public static String format(final String format, Object... args) {
+
```

Comment:
Those empty lines on start of the method are catching the eye. If we will take same methid JDK as an example, they don't follow that apporach, as well as most of the codebase

https://hg.openjdk.java.net/jdk/jdk12/file/06222165c35f/src/java.base/share/classes/java/lang/String.java#l2987
```
    public static String format(String format, Object... args) {
        return new Formatter().format(format, args).toString();
    }

```

Reply:
Will remove to comply with the style

Reply:
I think I removed all the ones I introduced, maybe even the ones that I didn't.